docs/source/verdi/verdi_user_guide.rst

```diff
@@ -26,155 +48,108 @@ For example ``verdi process kill --help`` will display:
 
-  * A usage line describing how to invoke and the accepted parameters
-  * A description of the commands functionality
+  * A ``Usage:`` line describing how to invoke the command
+  * A description of the command's functionality
   * A list of the available options
 
-The ``[OPTIONS]`` and ``[PROCESSES]...`` tags in the usage description, denote the 'parameters' that the command takes.
-A more detailed description of the available options will be listed after the description of the commands functionality
-The positional arguments will only be described in the command description itself.
-For a more detailed explanation of the difference between options and arguments, see the section about command line :ref:`parameters<cli_parameters>`.
+The ``Usage:`` line encodes information on the command's parameters, e.g.:
 
+ * ``[OPTIONS]``: this command takes one (or more) options
+ * ``PROCESSES``: this command requires a process as a positional argument
+ * ``[PROCESSES]``: this command takes a process as an optional positional argument
+ * ``[PROCESSES]...``: this command takes one or more processes as (optional) positional arguments
 
-.. _cli_parameters:
-
-Parameters
-----------
-Most of the ``verdi`` commands and their subcommands can take one or multiple parameters.
-In the language of command line interfaces, these parameters come in two flavors:
-
-  * Options
-  * Arguments
+Multi-value options are followed by ``...`` in the help string and the ``Usage:`` line
+of the corresponding command will contains the 'endopts' marker. For example::
 
-Arguments are positional parameters, whereas options are indicated by a flag that precedes the value, typically of the form ``-f``, or ``--flag``.
-The command line :ref:`help string<cli_help_strings>` section explained that each command will have a help string with a usage line, for example::
-
-  Usage: verdi process kill [OPTIONS] [PROCESSES]...
+  Usage: verdi export create [OPTIONS] [--] OUTPUT_FILE
 
-The ``[OPTIONS]`` tag indicates that the command takes one or multiple options and one or multiple ``[PROCESSES]`` as arguments.
-The square brackets in the usage line, indicate that the parameter is optional and not required.
-Three dots ``...`` following a parameter indicate that it can not take just one, but also more than one values.
+    Export various entities, such as Codes, Computers, Groups and Nodes, to an
+    archive file for backup or sharing purposes.
 
+  Options:
+    -X, --codes CODE...             one or multiple codes identified by their
+                                    ID, UUID or label
+    -Y, --computers COMPUTER...     one or multiple computers identified by
+                                    their ID, UUID or label
+    -G, --groups GROUP...           one or multiple groups identified by their
+                                    ID, UUID or name
+    -N, --nodes NODE...             one or multiple nodes identified by their ID
+                                    or UUID
```

Comment:
There are more options here for `verdi export create`, could you show this perhaps by having a line like "`...`" or explicitly stating that this is a part of the full help response?